Gemfile.lock

```diff
@@ -45,10 +45,10 @@ GEM
     database_cleaner (1.5.3)
     diff-lcs (1.3)
     erubis (2.7.0)
-    factory_girl (4.7.0)
+    factory_girl (4.8.0)
       activesupport (>= 3.0.0)
-    factory_girl_rails (4.7.0)
-      factory_girl (~> 4.7.0)
+    factory_girl_rails (4.8.0)
```

Comment:
@kheppenstall Out of curiosity, how come factory_girl_rails needed to be updated?  I have nothing against it - just curious what lead to updating the gem.